Tests/UnitTests/TestHelpers/AsyncTestHelpers.swift

```diff
@@ -88,3 +98,7 @@ func asyncWait(
         throw ConditionFailedError()
     }
 }
+
+// Higher value required to avoid slow CI failing tests.
+let defaultTimeout: DispatchTimeInterval = .seconds(2)
```

Comment:
Also increased the default timeout we were using from 1 second to 2, to avoid flaky failures when CircleCI is being slow.